### PR TITLE
test: Remove unused sanitizer suppressions

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: test/sanitizer_suppressions/ubsan [hunk 1]
```diff
@@ -5,9 +5,6 @@
 # names can be used.
 # See https://github.com/google/sanitizers/issues/1364
 
-# https://github.com/bitcoin/bitcoin/pull/21798#issuecomment-829180719
-signed-integer-overflow:policy/feerate.cpp
```
Comment (hebasto) [inline test/sanitizer_suppressions/ubsan:9]: > was fixed in gcc-11 and later.

Mind pointing at the actual fix if it is known?

**Review by hebasto — APPROVED** [review]
ACK 9999813e91b94e78ed65130478d9dcb6d787514f, I have reviewed the code and it looks OK, I agree it can be merged.

**Comment by fanquake** [comment]
Might as well have a go

**Review by fanquake — APPROVED** [review]
ACK fa5af4d510a8f129157db8b69a9a387036efb206

**Review by hebasto — APPROVED** [review]
ACK fa15a9934ee1d331737c631e6ffc2ddfafaddb7f

Nice to see the updated comments with TSan logs.

**Review by fanquake — APPROVED** [review]
ACK fa15a9934ee1d331737c631e6ffc2ddfafaddb7f

**Pull request merged**